test/helpers.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+package test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+func CreateFileWithErrorCheck(t *testing.T, dir, fileName string) *os.File {
```

Comment:
The `CreateFileWithErrorCheck` and `RemoveFileWithErrorCheck` functions seem a bit of an overkill to me. Don't think we need to make our own functions for creating files and directories.

Reply:
it simplifies 2 lines of code that appears numerous times in the tests

Reply:
I guess this is used a far bit alright in the test code. Just want to be careful that we dont overuse this pattern and end up with a lot of wrapper test functions that only check errors returned from another function.